servers/ups-wildfly/src/main/webapp/WEB-INF/hornetq-jms.xml

```diff
@@ -82,5 +82,5 @@
                 <entry name="/queue/AllBatchesLoadedQueue"/>
             </jms-queue>
         </jms-destinations>
-    </hornetq-server>
+    </server>
 </messaging-deployment>
```

Comment:
this is specific to AMQ Artemis - we should set queues with CLI script (same as we do with address-settings) and thus we will have two cli scripts (one for HornetQ, one for AMQ)... at the end, we can remove hornetq-jms.xml